service/hook/github/github.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/bitrise-io/go-utils/sliceutil"
 )
 
+const (
+	envKeyRepositoryOriginURL = "REPOSITORY_ORIGIN_URL"
```

Comment:
The env should start with BITRISE and maybe include webhook in it. It could be BITRISE_WEBHOOK_REPOSITORY_URL.